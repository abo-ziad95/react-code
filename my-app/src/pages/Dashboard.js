import React from 'react';
import Dropdown from "../coponents/Dropdown";
import {
  useParams
} from 'react-router-dom'
const Dashboard = () =>  {
  let { service_slug, brand_slug, style_slug } = useParams();
  return (
    <>
      <Dropdown match={service_slug} name="terms"/>
      <Dropdown match={brand_slug} name="brands_terms"/>
      <Dropdown match={style_slug}  name="styles"/>
    </>
  );
}

export default Dashboard;
