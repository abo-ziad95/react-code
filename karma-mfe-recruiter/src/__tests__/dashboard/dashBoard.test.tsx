import React from 'react';
import renderer from 'react-test-renderer';
import Karma from "@hatech/karma-core";
import Dashboard from "../../views/dashboard/dashboard";

it("Dashboard component renders correctly", () => {
    const dashBoard = renderer
        .create(<Karma><Dashboard/></Karma>)
        .toJSON();
    expect(dashBoard).toMatchSnapshot();
});