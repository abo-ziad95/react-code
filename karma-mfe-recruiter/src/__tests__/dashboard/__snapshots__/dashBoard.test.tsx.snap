// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-4 MuiGrid-grid-xl-4"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            >
              Organizations
            </span>
            <span
              className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
            >
              Number of organizations: 0
            </span>
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
        <div>
          Loading Chart
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-8 MuiGrid-grid-lg-8 MuiGrid-grid-xl-8"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            >
              Organizations Info
            </span>
            <span
              className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
            >
              Active organizations
            </span>
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
        <div>
          Loading Chart
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6 MuiGrid-grid-lg-6 MuiGrid-grid-xl-6"
    >
      <div
        style={
          Object {
            "paddingTop": 12,
          }
        }
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
        >
          <div
            className="MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Candidates
              </span>
              <span
                className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
              >
                Number of applied candidates: 0
              </span>
            </div>
          </div>
          <hr
            className="MuiDivider-root"
          />
          <div>
            Loading Chart
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-6 MuiGrid-grid-lg-6 MuiGrid-grid-xl-6"
    >
      <div
        style={
          Object {
            "paddingTop": 12,
          }
        }
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
        >
          <div
            className="MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Jobs
              </span>
              <span
                className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
              >
                Number of posted jobs: 0
              </span>
            </div>
          </div>
          <hr
            className="MuiDivider-root"
          />
          <div>
            Loading Chart
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
